### PR TITLE
Fixes dependency versions in orc8r

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Fixes dependency version in orc8r

## Checklist

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added tests that validate the behaviour of the software.
- [x] I validated that new and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I have bumped the version of any required library.

### CODE REVIEW

File: orc8r-certifier-operator/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,8 @@
...
-cryptography
-jsonschema
+ops==2.2.0
+ops-lib-pgsql==1.4
+psycopg2-binary==2.9.6  # Package only used for testing, charm uses charm-binary-python-packages from charmcraft.yaml
+lightkube==0.12.0
+lightkube-models==1.26.0.4
+jsonschema==4.17.3
+cryptography==40.0.2
+
```
Comment (sanchezfdezjavier) [inline orc8r-certifier-operator/requirements.txt]:
```suggestion
```

File: orc8r-nginx-operator/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,6 @@
-Jinja2
-jsonschema
-ops
-lightkube
-lightkube-models
+ops==2.2.0
+ops-lib-pgsql==1.4
```
Comment (sanchezfdezjavier) [inline orc8r-nginx-operator/requirements.txt]: Do we need this if not using `orc8r_base_db`?

**Review by sanchezfdezjavier — COMMENTED** [review]
I've seen there's version `1.24` for nginx, I don't if it's worth updating the rock. Probably it's not the priority right now.

File: orc8r-nginx-operator/requirements.txt (hunk 2 above)
Comment (saltiyazan) [reply orc8r-nginx-operator/requirements.txt]: Not in `nginx`, you are right, removed.

**Comment by saltiyazan** [comment]
> I've seen there's version `1.24` for nginx, I don't if it's worth updating the rock. Probably it's not the priority right now.

I'm not sure updating the rock is in the scope of this change or if it's even in the scope of the release.
What do you think?

**Review by Gmerold — APPROVED** [review]

**Review by gruyaume — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Comment by sanchezfdezjavier** [comment]
> > I've seen there's version `1.24` for nginx, I don't if it's worth updating the rock. Probably it's not the priority right now.
> 
> 
> 
> I'm not sure updating the rock is in the scope of this change or if it's even in the scope of the release.
> 
> What do you think?

I guess it's not. I just wanted to make sure we were aware.

**Pull request merged**